client/components/Nav.jsx

```diff
@@ -531,11 +531,19 @@ class Nav extends React.PureComponent {
         { __process.env.LOGIN_ENABLED && !this.props.user.authenticated &&
           <ul className="nav__items-right" title="user-menu">
             <li className="nav__item">
-              <p>
-                <Link to="/login">Log in</Link>
-                <span className="nav__item-spacer">or</span>
-                <Link to="/signup">Sign up</Link>
-              </p>
+              <Link to="/login">
```

Comment:
hey, so this html is not semantically correct—rather than inserting a `<button>` within the link, the styles for these should be updated.

Reply:
Got it, I'll keep this in mind and make changes.